src/index.tsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
```

Comment:
Unnecessary comma.

Reply:
It is not unnecessary, look at `.prettierrc`, you'll find this:
`"trailingComma": "all",`
It gives you better diffing in Git, when you change object properties etc.
Thx for approve.